src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![cfg(any(target_os = "macos", target_os = "ios"))]
+// #![cfg(any(target_os = "macos", target_os = "ios"))]
```

Comment:
Oops, I think I forgot to drop this even after cross-compiling/building/testing to the right target :sweat_smile: